main.js

```diff
@@ -0,0 +1,43 @@
+function createCalendar(id, year, month) {
+  const container = document.getElementById(id);
+  const currentMonth = month - 1;
+  const date = new Date(year, currentMonth);
+  const days = ['Пн', 'Вт', 'Ср', 'Чт', 'Пт', 'Сб', 'Нд'];
+  let table = '<table><tr>';;
+
+  for (let i = 0; i < days.length; i++) {
+    table += `<th>${days[i]}</th>`
+  }
+  table += '</tr><tr>';
```

Comment:
looks like a torn logic, here you close previous row and start next, will be better to extract functions like `createRow/createWeek`, `createCell/createDay` etc.